app/assets/styles/themes/census/_theme_vars.scss

```diff
@@ -1,5 +1,5 @@
 $color-purple: #6e2585;
 $color-secondary: $color-purple;
-$color-emphasis: #f1e9f3;
+$color-emphasis: #dce5b0;
```

Comment:
You can probably just remove this var altogether and it will be the same as the default theme—this file just overrides it.